__tests__/api.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection")
+const request = require("supertest")
+const app = require ("../app")
+const data = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed")
+
+beforeEach(()=> seed(data))
+afterAll(()=> db.end())
+
+
+describe("API tests", ()=>{
+    describe("GET api/topics", ()=>{
+        test("responds with 200 status code", () => {
+            return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({body}) => {
+                    body.rows.forEach((topic) => {
```

Comment:
Ideally we should check length of array if asserting inside a forEach, to ensure we get all topics back.